.github/workflows/bot.yml

```diff
@@ -570,11 +575,11 @@ jobs:
       matrix:
         include:
           - scalaProfile: 'scala-2.13'
-            flinkProfile: 'flink1.18'
+            flinkProfile: 'flink1.19'
```

Comment:
For bundle validation, we also need to update the docker image release for Flink: https://github.com/apache/hudi/blob/d4a4d9c43e40a4d16b7a4859bd19b3704e5794c1/packaging/bundle-validation/ci_run.sh#L43 which the bundle jar would be run upon.

Reply:
I see. Thanks for the hint.